sdk/ml/azure-ai-ml/tests/dsl/unittests/test_command_builder.py

```diff
@@ -162,6 +158,8 @@ def test_command_function(self, test_command):
                 "tags": {},
             }
         }
+        a = test_command._component._to_rest_object()
```

Comment:
what is this for?

Reply:
Good catch, must be debug lines forgot to delete